ui/src/app/views/workspace-edit/component.html

```diff
@@ -173,6 +173,9 @@ <h3>Disease focused research</h3>
                 </div>
                 <div>
                   <label class="checkbox-label" (click)="workspace.researchPurpose.containsUnderservedPopulation = !workspace.researchPurpose.containsUnderservedPopulation"><h3>Includes an underserved population</h3></label>
+                  <div>
+                    This research will focus on or include findings on distinguishing characteristics related to one or more underserved populations
```

Comment:
Nit: I think this could use commas, but won't hold up the PR. Consider:
```
This research will focus on, or include findings on, distinguishing characteristics related to one or more underserved populations
```
Also ... question ... where does this language come from? As a matter of process (that I wouldn't impose upon the team, but will gently recommend), I like to see some form of documentation behind PRs - usually in a jira ticket.

Reply:
This comes from a conversation with Moran. Agreed that we should probably make a ticket for the future.